cli/command/container/utils_test.go

```diff
@@ -63,7 +63,7 @@ func TestWaitExitOrRemoved(t *testing.T) {
 
 	client := test.NewFakeCli(&fakeClient{waitFunc: waitFn, Version: api.DefaultVersion})
```

Comment:
Oh, I guess I can potentially remove the `NewFakeCli` wrapper here; let me check.

Reply:
yup! removed.